### ISSUE TITLE
Dynamic imports not extracted to chunks

### ISSUE THREAD

**Issue by josuesan** [vercel/next.js#14274]
# Dynamic imports not extracted to chunks

## Description
I am trying to dynamically import a component to remove it from the main bundle of my application, but despite compiling successfully the bundle remains exactly the same and does not generate a chunk "1.js" with the dynamic section that it exports.

## Expected behavior

At the moment of checking the network of the browser console I hope to find a chunk with the imported section of my code dynamically

## Screenshots
![image](https://user-images.githubusercontent.com/26355477/84921145-173fea80-b092-11ea-8e08-e3fe845be61e.png)
Before Dynamic component added.

![image](https://user-images.githubusercontent.com/26355477/84921001-e5c71f00-b091-11ea-8ef6-c9e28712d871.png)
Dynamic Component

![image](https://user-images.githubusercontent.com/26355477/84921253-3c345d80-b092-11ea-8c72-824e37277950.png)
After dynamic component added the bundle size has the same size and chunk 1.js dont appear

## System information

- OS: Windows
- Browser (Chrome
- Version of Next.js: 9.4.4
- Version of Node.js: 10.16.2

**Comment by jamesmosier** [comment]
Do you happen to have a minimal reproduction for this?

**Comment by josuesan** [comment]
I managed to solve the error. I have an index.js file in which I export all the components of my application there (there was also a component to which I was doing Dynamic import).

![image](https://user-images.githubusercontent.com/26355477/85653443-0e937980-b67b-11ea-8cf1-a804e9023683.png)
List of my export components

When I excluded it from the list of the components it exported, the chunk of the dinamyc started to be generated.

I infer that next already had that component in the main bundle of the application and when I tried to do dinamyc import it did not work because that component was already included in the bundle. This makes sense doesn't it?

Thank you, anyway, for the time invested
